plugins/zenoh-plugin-storage-manager/src/replica/snapshotter.rs

```diff
@@ -268,11 +278,17 @@ impl Snapshotter {
             if ts > *last_snapshot_time {
                 remains_volatile.insert(k, ts);
             } else {
-                let deleted = stable.insert(k, ts);
+                let deleted = stable.insert(k.clone(), ts);
                 if deleted.is_some() {
```

Comment:
Same comment than above: better to replace `if x.is_some() { ... x.unwrap() ... }` 
with `if let Some(y) = x { ... }`